jobs/exportCovid19.ts

```diff
@@ -12,7 +12,7 @@ export async function exportCovid19(done, horas) {
     const pipelineVacunaCovid19 = [
         {
             $match: {
-                'solicitud.fecha': {
+                'estadoActual.updatedAt': {
```

Comment:
Los estados no tiene fecha updatedAt. Hay que usar la fecha de ejecución y armar un indice. 

```suggestion
                'ejecucion.fecha': {
```

Reply:
cuando pasa de ejecución a validada vi que se generaba un updatedAt. Pero no en el array de estados sino dentro de estado actual.